.github/pull_request_template.md

```diff
@@ -20,20 +21,8 @@ not fully covered by unit tests or manual testing can be complicated. -->
 
 <!-- Put an 'x' in all the boxes that apply -->
 
-- [ ] I have added unit tests to cover my changes.​
-- [ ] I have added integration tests to cover my changes.​
-- [ ] I have ran e2e tests and there is no issues.
-- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/open-edge-platform/training_extensions/blob/develop/CHANGELOG.md) in develop).​
-- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/open-edge-platform/training_extensions/tree/develop/docs) in develop).
-- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
-
-### License
-
-- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/open-edge-platform/training_extensions/blob/develop/LICENSE) that covers the project.
-      Feel free to contact the maintainers if that's a concern.
-- [ ] I have updated the license header for each file (see an example below).
-
-```python
-# Copyright (C) 2025 Intel Corporation
-# SPDX-License-Identifier: Apache-2.0
-```
+- [ ] The PR title and description are clear and descriptive
+- [ ] I have manually tested the changes
+- [ ] All changes are covered by automated tests
+- [ ] All related issues are linked to this PR (if applicable)
+- [ ] Documentation has been updated (if applicable)
```

Comment:
Maybe later, but ideally even these should be automated.

I've just merged a similar template on geti action 
https://raw.githubusercontent.com/open-edge-platform/geti-action/refs/heads/main/.github/pull_request_template.md